File: Structural/Bridge/Bridge_py/GenericDevice.py
import abc

class GenericDevice(metaclass=abc.ABCMeta):
    def __init__(self):
        self.deviceState=None
        self.maxSetting=None
        self.volumeLevel=0

    @abc.abstractmethod
    def buttonFivePressed(self):
        pass

    @abc.abstractmethod
    def buttonSixPressed(self):
        pass

    def buttonSevenPressed(self):
        self.volumeLevel+=1
        print("VolumeLevel:"+str(self.volumeLevel))

    def buttonEightPressed(self):
        self.volumeLevel-=1
        print("VolumeLevel:"+str(self.volumeLevel))

    def deviceFeedback(self):
        if self.deviceState>self.maxSetting or self.deviceState<0:
            print("On "+self.deviceState)

File: Structural/Bridge/Bridge_Java/GenericDevice.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Bridge;

/**
 *
 * @author sashant
 */
public abstract class GenericDevice {
    public int deviceState;
    public int maxSetting;
    public int volumeLevel=0;
    
    public abstract void buttonFivePressed();
    public abstract void buttonSixPressed();
    
    public void deviceFeedback(){
        if(this.deviceState>this.maxSetting || this.deviceState<0){
            System.out.println("On "+deviceState);
        }
    }
    
    public void buttonSevenPressed(){
        volumeLevel++;
        System.out.println("Volume:"+volumeLevel);
    }
    
    public void buttonEightPressed(){
        volumeLevel--;
        System.out.println("Volume:"+volumeLevel);
    }
    
}
